h/static/styles/components/_share-widget.scss

```diff
@@ -145,6 +145,8 @@
   text-decoration: none;
   font-size: 20px;
   cursor: pointer;
+  padding-left: 5px;
+  padding-right: 5px;
```

Comment:
This CSS is legacy, so I'm inclined to just say leave this as it lies. In our more modern CSS, I would favor `margin` here and would probably apply it to one side only and use a pseudo-class selector to exclude the margin from the first or last of the icons. Here, that would probably be more trouble than it's worth.